scripts/hot_cold.py

```diff
@@ -0,0 +1,105 @@
+"""
+This is a sample simulation that does not represent any particular biological system. It is just a showcase 
+of how create a Simulation object, add forces, and initialize the reporter. 
+
+In this simulation, a simple polymer chain of 10,000 monomers is 
+"""
+import time
+import numpy as np
+import os, sys
+import polychrom
+from polychrom import simulation, starting_conformations, forces, forcekits
+from polychrom.integrators import ActiveBrownianIntegrator
+import openmm
+from polychrom.hdf5_format import HDF5Reporter
+from simtk import unit
+from pathlib import Path
+
+total_runs = 200
+runs_per_gpu = total_runs // 4
+
+def run_sim(gpuid, run_number, activity_ratio, timestep=170, ntimesteps=200000, blocksize=100):
+    """ Run a single simulation on GPU i."""
+    ids = np.load('/net/levsha/share/deepti/data/ABidentities_blobel2021_chr2_35Mb_60Mb.npy')
+    N=len(ids)
```

Comment:
I would consider adding these files to the repository in some examples folder. They won't work for people who are not in the mirny lab.

Reply:
whoops this file wasn't supposed to be part of the PR. removed it